Cargo.toml

```diff
@@ -35,7 +35,7 @@ libc = "0.2"
 rustc-serialize = "0.3"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
-zeroize = "0.9"
+zeroize = { version = "1.0", features = ["zeroize_derive"] }
```

Comment:
Can update to latest subversion (`1.3.x`), but it comes with a MSRV bump to Rust `1.47`

`zeroize` versions - MSRV

- 1.0.0 - 1.36
- 1.1.0 - 1.39
- 1.1.1 - 1.39
- 1.2.0 - 1.44
- 1.3.0 - 1.47